### DISCUSSION TITLE
eval_prompts getting cut?

### DISCUSSION THREAD

**Discussion by trialan** [CarperAI/trlx#195] [Q&A] [answered]
Is there somewhere in the code that limits the size of eval prompts? I'm using quite large eval prompts (500+ tokens) and in the eval tables I noticed that they are not being used as I expected (truncated to the right). Is there some sort of max size? 

looking at eval tables the size they get truncated to is not constant but is around 420 to 450 tokens

EDIT:
I can see in the summarize_rlhf example there's something about cutting summaries at 550 tokens.

**Comment by LouisCastricato** [comment, marked as answer]
Please make an issue, not a discussion.
